air-quality-ui/system_tests/ui/city_page.spec.ts

```diff
@@ -61,6 +57,43 @@ test('Legend deselect removes chosen site', async ({ page, cityPage }) => {
   await expect(cityPage.textFinder('Centro')).not.toBeVisible()
 })
 
+test.describe('City page resolution tests', () => {
+  ;[
+    {
+      width: 960,
```

Comment:
can you put a comment in the test data for each describing what the data represents, eg potential laptop dimensions